model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb

```diff
@@ -0,0 +1,629 @@
+    "## Setup environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "K3DOvHnXsf-I",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Install itk for DICOM I/O and for reading DICOM into an itkImage \n",
+    "#   that manages all DICOM field values, include acquistion details \n",
+    "#   such as voxel image, image orientation, and image directions,\n",
+    "#   which are critical to image processing and display.\n",
+    "\n",
+    "# Upgrade pip, just in case.\n",
+    "!python -m pip install --upgrade -q pip\n",
+    "\n",
+    "# installations required for monai\n",
+    "!python -c \"import monai\" || pip install -q \"monai[nibabel,itk,tqdm,pandas,skimage]\"\n",
+    "\n",
+    "# These are the libraries used to read DICOM Seg objects.\n",
+    "!python -m pip install -q pydicom pydicom-seg\n",
+    "\n",
+    "# Install tcia_utils to download the datasets.\n",
+    "!python -m pip install --upgrade -q tcia_utils\n",
+    "\n",
+    "# This is the installation required for itkWidgets.\n",
+    "!python -m pip install --upgrade --pre -q \"itkwidgets[all]==1.0a20\" imjoy_elfinder"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "tF8-MqoN6YT5"
+   },
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "\n",
+    "# Numpy for numpy.arrays\n",
+    "import numpy as np\n",
+    "\n",
+    "# Include ITK for DICOM reading.\n",
+    "import itk\n",
+    "\n",
+    "# Include pydicom_seg for DICOM SEG objects\n",
+    "import pydicom\n",
+    "import pydicom_seg\n",
+    "\n",
+    "# for downloading data from TCIA\n",
+    "from tcia_utils import nbia\n",
+    "\n",
+    "# This is the most common import command for itkWidgets.\n",
+    "#   The view() function opens an interactive viewer for 2D and 3D\n",
+    "#   data in a variety of formats.\n",
+    "from itkwidgets import view\n",
+    "\n",
+    "# imports for monai\n",
+    "import torch\n",
+    "from monai.data import decollate_batch\n",
+    "from monai.bundle import ConfigParser, download\n",
+    "\n",
+    "from monai.config import print_config\n",
+    "\n",
+    "print_config()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Ub0GzFRXemJN"
+   },
+   "outputs": [],
+   "source": [
+    "# If running on SageMaker or Studio Lab, install essential packages and extensions.\n",
+    "if \"studio-lab-user\" in os.getcwd():\n",
+    "    print(\"Upgrading dependencies\")\n",
+    "    !conda install --yes -q --prefix {sys.prefix} -c conda-forge opencv nodejs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BctRdoEcsf-G"
+   },
+   "source": [
+    "### On many systems you must manually install the imjoy-jupyter-extension for itkWidgets!\n",
```

Comment:
I think the reader may want to read this markdown section after they install the imjoy extension and before they run the imports. How about putting this with the "## Setup environment " markdown block, for example:
```markdown
## Setup environment
NOTE: On many systems you must manually install the imjoy-jupyter-extension for itkWidgets!
If you do not see a blue 'ImJoy' icon on the menu bar in this notebook:

Google CoLab: The following does not apply to Google CoLab - it will not show an ImJoy and all should work without modification.
Enable Extensions: Many Jupyter Lab systems disable jupyter extensions by default, and they must be enabled for this notebook to work. Typically this is accomplished using the Jupyter interface to select the extension manager (left-hand side, icon that looks like a piece of a puzzle) and select the Enable button if it appears.
Install imjoy extension: In the extension manager, search for 'imjoy' and install the 'imjoy-jupyter-extension'. The installation can take several minutes. It may also prompt you to rebuild, save, and reload your jupyter environment as part of this process. In the end, you should see a blue 'ImJoy' icon on the left side of the menu bar in this notebook.
```

Reply:
I think it'd be good for Stephen to chime in here too.

Reply:
thanks I moved this cell up to the #Setup section.